build.sbt

```diff
@@ -281,10 +280,14 @@ lazy val appCommonsTest = project
 lazy val oracleServer = project
   .in(file("app/oracle-server"))
   .settings(CommonSettings.prodSettings: _*)
+  .settings(
+    Compile / unmanagedResourceDirectories += baseDirectory.value / "src" / "universal"
```

Comment:
This is where the custom configuration is stored and added to the resources packaged in the artifact.